start-site/src/main/resources/application.yml

```diff
@@ -1142,32 +1160,49 @@ initializr:
               href: https://cloud.spring.io/spring-cloud-static/spring-cloud-circuitbreaker/current/reference/html
         - name: Hystrix [Maintenance]
           id: cloud-hystrix
-          description: Circuit breaker with Spring Cloud Netflix Hystrix.
+          description: "Circuit breaker with Spring Cloud Netflix Hystrix. Maintenance mode, suggested replacement: Spring Cloud CircuitBreaker."
           groupId: org.springframework.cloud
           artifactId: spring-cloud-starter-netflix-hystrix
           compatibilityRange: "[2.0.0.RELEASE,2.4.0-M1)"
           links:
             - rel: guide
               href: https://spring.io/guides/gs/circuit-breaker/
               description: Circuit Breaker
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-netflix/docs/2.2.6.RELEASE/reference/html/#circuit-breaker-hystrix-clients
         - name: Hystrix Dashboard [Maintenance]
           id: cloud-hystrix-dashboard
-          description: Circuit breaker dashboard with Spring Cloud Netflix Hystrix.
+          description: Circuit breaker dashboard with Spring Cloud Netflix Hystrix
```

Comment:
For consistency, should this note that it's in maintenance mode too? Is there a suggested replacement?

Reply:
There's no direct suggested replacement, (i.e., we suggest using Micrometers + a metrics aggregation tool, but have discussed it with @mbhave and we have reached the conclusion that it's not straightforward enough to include it here).